packages/arb-tx-aggregator/web3/eth.go

```diff
@@ -410,6 +420,9 @@ func (s *Server) makeTransactionResult(res *evm.TxResult) (*TransactionResult, e
 		V:                (*hexutil.Big)(vVal),
 		R:                math.U256Bytes(rVal),
 		S:                math.U256Bytes(sVal),
+		L1SeqNum:         (*hexutil.Big)(provenance.L1SeqNum),
+		ParentRequestId:  parentRequestId,
+		IndexInParent:    (*hexutil.Big)(provenance.IndexInParent),
```

Comment:
Is the IndexInParent the same value as returned in the TransactionIndex [in the transaction receipt](https://github.com/OffchainLabs/arbitrum/blob/b6856893221c388f84b115bf118bff2078027483/packages/arb-tx-aggregator/web3/eth.go#L254)?

Reply:
Nah, this is the index in the arbitrum batch